libraries/adaptive-expressions/tests/badExpression.test.ts

```diff
@@ -0,0 +1,546 @@
+import { ExpressionParser } from '../src';
+import assert from 'assert';
+
+const parser = new ExpressionParser();
+
+const scope = {
+    one: 1.0,
+    two: 2.0,
+    hello: 'hello',
```

Comment:
Do we really need both a .js and .ts test file?  If so, then you should factor out all of the tests into a common file.  Otherwise adding a new test requires adding it in two places which is likely to get missed.

Reply:
We don't, this was an oversight on my part. I started moving all the tests to Typescript but quickly decided against it as the rest of the packages don't. These files got tracked and were never removed. I'll remove them.